### ISSUE TITLE
format_parameters for log()

### ISSUE THREAD

**Issue by strengejacke** [easystats/parameters#128]
There is currently no bug, but maybe we have a good idea, if we can improve the formatting for, say, log-transformation?

But maybe it's already the best solution, I just wanted to do some "brain storming".

``` r
model <- lm(Sepal.Length ~ Species + log(Sepal.Width/10), data = iris)
parameters::format_parameters(model)
#>            (Intercept)      Speciesversicolor       Speciesvirginica 
#>          "(Intercept)" "Species (versicolor)"  "Species (virginica)" 
#>    log(Sepal.Width/10) 
#>  "log(Sepal.Width/10)"
```

<sup>Created on 2019-09-26 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by DominiqueMakowski** [comment]
Mmh you think something like `Sepal.Width/10 (log)` would be better?

**Comment by strengejacke** [comment]
Not sure, probably the best way is to keep it as it is... But probably we can remove `I()`?

``` r
model <- lm(Sepal.Length ~ Species + I(Sepal.Width/10), data = iris)
parameters::format_parameters(model)
#>            (Intercept)      Speciesversicolor       Speciesvirginica 
#>          "(Intercept)" "Species (versicolor)"  "Species (virginica)" 
#>      I(Sepal.Width/10) 
#>    "I(Sepal.Width/10)"
```

<sup>Created on 2019-09-27 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by strengejacke** [comment]
I think this can be closed along with #129
